parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -5063,3 +5065,46 @@ SqlCloseCursor SqlCloseCursor() :
     <CLOSE> cursorName = SimpleIdentifier()
     { return new SqlCloseCursor(s.end(this), cursorName); }
 }
+
+SqlSelectAndConsume SqlSelectAndConsume() :
```

Comment:
Would it ever be valid to call this outside of CursorStmt()? If not, I think it would make sense to add a top-level comment to this function clarifying in which contexts this type of syntax is valid.

Reply:
I was looking at the docs again and there is a form of SELECT AND CONSUME that can be used standalone but it doesn't allow the INTO clause. I could add a comment here clarifying this SELECT format is only used inside stored procedures

Reply:
SGTM.